src/makeDOMDriver.ts

```diff
@@ -62,8 +63,13 @@ function makeDOMDriver(container: string | Element, options?: DOMDriverOptions):
     /* tslint:disable:no-empty */
     rootElement$.addListener({next: () => {}, error: () => {}, complete: () => {}});
     /* tslint:enable:no-empty */
-
-    return new DOMSource(rootElement$, runStreamAdapter, [], isolateModule, delegators);
+    const source = new DOMSource(rootElement$, runStreamAdapter, [], isolateModule, delegators);
+    const _dispose = source.dispose;
+    source.dispose = () => {
+      clean$.shamefullySendNext('');
+      _dispose.apply(source);
+    };
```

Comment:
Not nice type of monkey patching with mutation. We can put clean$ as a member of DOMSource.